### ISSUE TITLE
[pimatic-log-reader] LogWatcher : ... Property "xLink" is not a valid property

### ISSUE THREAD

**Issue by Yves911** [pimatic/pimatic#958]
Hello 
using a xLink into LogWatcher generate an error message : Property "xLink" is not a valid property
here is my config:
```
    {
      "id": "bourseca",
      "name": "CA",
      "class": "LogWatcher",
      "file": "/dd_extern/tmp/bourse.log",
      "xLink": "http://www.boursier.com/actions/cours/credit-agricole-FR0000045072,FR.html",
      "attributes": [
        {
          "name": "Achat",
          "type": "number",
          "unit": "€"
        },
        {
          "name": "Perf",
          "type": "number",
          "unit": "€"
        },
        {
          "name": "Cours",
          "type": "number",
          "unit": "€"
        },
        {
          "name": "Variation",
          "type": "string"
        }
      ],
      "lines": [
        {
          "match": "CA,(.*),(.*),(.*),(.*),",
          "Perf": "$1"
        },
        {
          "match": "CA,(.*),(.*),(.*),(.*),",
          "Achat": "$2"
        },
        {
          "match": "CA,(.*),(.*),(.*),(.*),",
          "Cours": "$3"
        },
        {
          "match": "CA,(.*),(.*),(.*),(.*),",
          "Variation": "$4"
        }
      ],
      "xAttributeOptions": [
        {
          "name": "Achat",
          "displaySparkline": false
        }
      ]
    },
```

The link is created and working fine the only issue is the error message at pimatic startup.
Let me know if you need additional details
Cheers
Yves

**Comment by mwittig** [comment]
Thanks for the report! I have filed a PR, but it may takes some time to get this processed as we're in the middle of the holiday season.

**Comment by Yves911** [comment]
Thanks

**Comment by mwittig** [comment]
published pimatic-log-reader@0.9.0

**Comment by Yves911** [comment]
Working fine thanks
